drupal/README.md

```diff
@@ -0,0 +1,78 @@
+# Drupal
+
+> Drupal is one of the most versatile open source content management systems on the market.
+
+Based on the [Bitnami Drupal](https://github.com/bitnami/bitnami-docker-drupal) image for docker, this Chart bootstraps a [Drupal](https://drupal.org/) deployment on a [Kubernetes](https://kubernetes.io) cluster using [Helm](https://helm.sh).
+
+## Dependencies
+
+The Drupal Chart requires the [Bitnami MariaDB Chart](https://github.com/bitnami/charts/tree/master/mariadb) for setting up a database backend.
+
+Please refer to the [README](https://github.com/bitnami/charts/tree/master/mariadb) of the Bitnami MariaDB Chart for deployment instructions.
+
+## Persistence
+
+> *You may skip this section if your only interested in testing the Drupal Chart and have not yet made the decision to use it for your production workloads.*
+
+For persistence of the Drupal configuration and user file uploads, mount a [storage volume](http://kubernetes.io/v1.0/docs/user-guide/volumes.html) at the `/bitnami/drupal` path of the Drupal pod.
+
+By default the Drupal Chart mounts an [emptyDir](http://kubernetes.io/docs/user-guide/volumes/#emptydir) volume.
+
+## Configuration
+
+To edit the default Drupal configuration, run
+
+```bash
+$ helm edit drupal
```

Comment:
`helm` is now `helmc` (helm classic). You need to replace all references to the command `helm` with `helmc`